runtime/js-compute-runtime/builtins/crypto-algorithm.cpp

```diff
@@ -13,6 +13,67 @@ namespace builtins {
 
 namespace {
 
+// Web Crypto API uses DOMExceptions to indicate errors
+// We are adding the fields which are tested for in Web Platform Tests
+// TODO: Implement DOMExceptions class and use that instead of duck-typing on an Error instance
```

Comment:
Functionally speaking, the only step missing here from this being a "true" DOMException is a `.__proto__ = globalThis.DOMException.prototype` addition, where the `DOMException` could otherwise be implemented in JS. Just wondering if it might be a relatively simple addition to support the proper inheritance, even if it is just duck typing for now.